bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -45,48 +50,15 @@ for fi in files:
 logging.info('combining foreground and foreground excluded segments')
 # Set up dictionaries to contain segments from the individual statmap files
 indiv_segs = segments.segmentlistdict({})
-indiv_segs_fg_exc = segments.segmentlistdict({})
-
-any_fg_vetoes = False
-# loop through statmap files and put segments into segmentlistdicts
-for fi in files:
-    key = fi.attrs['ifos'].replace(' ','')
-    # get analysed segments from individual statmap files
-    starts = fi['segments/{}/start'.format(key)][:]
-    ends = fi['segments/{}/end'.format(key)][:]
-    indiv_segs[key] = pycbc.events.veto.start_end_to_segments(starts, ends)
-    # get the foreground_vetoed times from individual statmap files
-    if 'foreground_veto' in fi['segments']:
-        any_fg_vetoes = True
-        remove_start_time = fi['segments/foreground_veto/start'][:]
-        remove_end_time = fi['segments/foreground_veto/end'][:]
-        vetoed_segs = pycbc.events.veto.start_end_to_segments(remove_start_time,
-                                                          remove_end_time)
-        # remove from analysis segments
-        indiv_segs_fg_exc[key] = indiv_segs[key] - vetoed_segs
 
 # Convert segmentlistdict to a list ('seglists') of segmentlists
 # then numpy.sum(seglists, axis=0) does seglists[0] + seglists[1] + ...
 foreground_segs = numpy.sum(list(indiv_segs.values()), axis=0)
```

Comment:
`indiv_segs` isnt getting set, as it still needs to go through the loop to get that from the files, so `foregroud_segs` wont have any segments involved, and `f.attrs['foreground_time']` will be zero